README.md

```diff
@@ -20,6 +20,13 @@ was written correctly and much more.
 
 ![Etcher](https://raw.githubusercontent.com/resin-io/etcher/master/screenshot.png)
 
+Supported Operating Systems
+---------------------------
+
+- GNU/Linux
+- macOS 10.9 and later
```

Comment:
@jhermsmeier Or you could PR a fix? "See something that needs fixing? Propose a change on the source."